pkg/webreg/webreg.go

```diff
@@ -844,6 +849,11 @@ The following environment variables will be available to commands in a step:
     <td>Image pull specification for the ephemeral release payload used to install the ephemeral OpenShift cluster.</td>
     <td>Always.</td>
   </tr>
+  <tr>
+    <td style="white-space: nowrap"><code>${LEASED_RESOURCE}</code></td>
+    <td>The name of the <i>resource</i> leased to grant access to cloud quota. See <a href="./help/leases">the documentation</a>.</td>
+    <td>When the test requires a <i>lease</i>.</td>
```

Comment:
I am always going to find the italic clunky to read, it looks like it was supposed to stress something when its supposed to indicate that we are talking about a specific thing we defined.

Its btw also not in line with what the openshift docs do, they do use code style for this situation, see e.G. here: https://docs.openshift.com/container-platform/4.3/operators/olm-adding-operators-to-cluster.html

Reply:
They seem to use bold, or code, or both. I'll sync with them on this one